### PR TITLE
fix(intl): remove extra slash from base-url

### PR DESCRIPTION
**Pull request by Francois-Esquire**
Removing trailing `/` from `baseUrl` when combined with `[baseUrl, locale, filename].join('/')`.

[Examples of `baseUrl`](https://github.com/americanexpress/one-app/search?q=baseUrl&unscoped_q=baseUrl)

### CODE REVIEW

File: src/intl/index.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ const getUrl = ({ getState, langPackLocale, componentKey }) => {
   const localeFilename = state.getIn(['config', 'localeFilename']);
 
   return `${[
-    moduleBaseUrl,
```
Comment (JAdshead) [inline src/intl/index.js:146]: can you include a test to cover this

Comment (mtomcal) [reply src/intl/index.js:146]: This may be easiest to test if you promote the `replace` to its own function.

Comment (Francois-Esquire) [reply src/intl/index.js:146]: Good call @mtomcal I'm going to try that.

Comment (Francois-Esquire) [reply src/intl/index.js]: updated with tests 6bc1c5f

**Pull request merged**